segmentstore/storage/src/test/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorageTests.java

```diff
@@ -2480,6 +2575,16 @@ public SegmentMetadata insertMetadata(String testSegmentName, long maxRollingLen
                 segmentMetadata.checkInvariants();
                 txn.create(segmentMetadata);
 
+                for (long blockStartOffset = 0; blockStartOffset < segmentMetadata.getLength(); blockStartOffset += config.getIndexBlockSize()) {
+                    val floor = index.floorEntry(blockStartOffset);
+                    txn.create(ReadIndexBlockMetadata.builder()
```

Comment:
It's hard to check here, but this doesn't return a future, does it?

Reply:
No it does not return future.